_templates/resources.html

```diff
@@ -29,6 +29,7 @@ <h2 id="directories"><img src="/img/icons/ico_invoice.svg" class="titleicon" alt
     <p><a href="https://99bitcoins.com/who-accepts-bitcoins-payment-companies-stores-take-bitcoins/">{% translate linkmerchants %}</a> - 99Bitcoins.com</p>
     <p><a href="https://www.buybitcoinworldwide.com/">{% translate linkexchanges %}</a> - buybitcoinworldwide.com</p>
     <p><a href="https://en.bitcoin.it/wiki/Category:Shopping_Cart_Interfaces">{% translate linkmerchantstools %}</a> - en.bitcoin.it</p>
+    <p><a href="http://www.bitcoinprojects.net/">BitcoinProjects.net</a></p>
```

Comment:
Could you please update this to match the formatting of the other entries?

Example:
[Projects](http://www.bitcoinprojects.net/) - bitcoinprojects.net